pkg/controller/nodeipam/ipam/cidrset/cidr_set_test.go

```diff
@@ -341,6 +343,77 @@ func TestCIDRSet_AllocationOccupied(t *testing.T) {
 	}
 }
 
+func TestDoubleOccupyRelease(t *testing.T) {
+	clusterCIDRStr := "10.42.0.0/16"
+	operations := []struct {
+		cidrStr     string
+		occupy      bool
```

Comment:
I think that is a bit difficult to understand the test with a boolean, I mean, It's not clear at first sight that occupy=false means release(). Maybe use a string operation string that can take the values "occupy" or "release"?